sdk/data/azcosmos/cosmos_global_endpoint_manager.go

```diff
@@ -143,15 +146,23 @@ func (gem *globalEndpointManager) GetAccountProperties(ctx context.Context) (acc
 		if err != nil {
 			return accountProperties{}, fmt.Errorf("failed to parse account properties: %v", err)
 		}
+		gem.once.Do(func() {
```

Comment:
Why once? Wouldn't it be beneficial to log whenever anything changes?

Reply:
we run update every 5 minutes so it's not necessarily every time something changes - but yeah that's why I raised this question in the description

Reply:
It might be relevant to log every time it changes, and if we cannot tell when that is, at least every time

Reply:
makes sense, will update this to log every time then